raw/GB/language-tables/Central_Veps.csv

```diff
@@ -1,196 +1,196 @@
-GB305,Is there a phonologically independent reflexive marker?,1,Grünthal 2015: 224,,GB305
-GB306,Is there a phonologically independent reciprocal marker that does not consist of two quantifiers (e.g. each + other,0,Grünthal 2015:in passim,,
-GB309,Are there multiple past or multiple future tenses, distinguishing distance from Time of Reference?,0,Grünthal 2015:128,259,,
-GB312,Is there overt morphological marking on the verb dedicated to mood?,1,Grünthal 2015: 141,,GB312
-GB313,Are there special adnominal possessive pronouns that are not formed by an otherwise regular process?,0,Grünthal 2015:105,,
-GB314,Can augmentative meaning be expressed productively by a shift of noun class/gender?,0,Grünthal 2015:in passim,,
-GB315,Can diminutive meaning be expressed productively by a shift of noun class/gender?,0,Grünthal 2015:in passim,,
-GB316,Is singular number regularly marked in the noun phrase by a phonologically free element?,0,Grünthal 2015:in passim,,
-GB317,Is dual number regularly marked in the noun phrase by a phonologically free element?,0,Grünthal 2015:in passim,,
-GB318,Is plural number regularly marked in the noun phrase by a phonologically free element?,0,Grünthal 2015:in passim,,
-GB319,Is trial number regularly marked in the noun phrase by a phonologically free element?,0,Grünthal 2015:in passim,,
-GB320,Is paucal number regularly marked in the noun phrase by a phonologically free element?,0,Grünthal 2015:in passim,,
-GB321,Is there a large class of nouns whose noun class/gender is not phonologically or semantically predictable?,0,Grünthal 2015:in passim,,
-GB322,Is there grammatical marking of direct evidence (perceived with the senses)?,0,Grünthal 2015:in passim,,
-GB323,Is there grammatical marking of indirect evidence (hearsay, inference, etc.)?,0,Grünthal 2015:in passim,,
-GB324,Is there an interrogative verb for content interrogatives (who?, what?, etc.)?,0,Grünthal 2015:224,,
-GB325,Is there a count/mass distinction in interrogative quantifiers?,0,Grünthal 2015:224,,
-GB326,Do (nominal) content interrogatives normally or frequently occur in situ?,0,Grünthal 2015:244–246,,
-GB327,Can the relative clause follow the noun?,1,Grünthal 2015: 302,,GB327
-GB328,Can the relative clause precede the noun?,1,Grünthal 2015: 302-303,,
-GB329,Are there internally-headed relative clauses?,0,Grünthal 2015:302-303,,
-GB330,Are there correlative relative clauses?,0,Grünthal 2015:302-303,,
-GB331,Are there non-adjacent relative clauses?,1,Grünthal 2015: 302,,GB331
-GB333,Is there a decimal numeral system?,1,Grünthal 2015: 246,,GB333
-GB334,Is there synchronic evidence for any element of a quinary numeral system?,0,Grünthal 2015:244–246,,
-GB335,Is there synchronic evidence for any element of a vigesimal numeral system?,0,Grünthal 2015:244–246,,
-GB336,Is there a body-part tallying system?,0,Grünthal 2015:in passim,,
-GB400,Are all person categories neutralized in some voice, tense, aspect, mood and/or negation?,0,Grünthal 2015: 121–126,,

```

Comment:
It looks like all feature titles are simply cut at the first comma. This cannot be intended, right?